### DISCUSSION TITLE
Rendering of wide jupyter notebook cells (UX)

### DISCUSSION THREAD

**Discussion by kopytjuk** [community/community#77448] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When looking at a notebook with a long line of code in a cell, depending on the screen width of the device, code disappears and a user does not see it. It is **not** possible to scroll horizontally (I am using Edge) 

High screen width (original):

<img width="987" alt="image" src="https://github.com/community/community/assets/12762439/88518dc6-98a0-4c69-a465-23874a46aeab">

Low screen width (some calculation invisible and leads to errors when copy & pasting):

<img width="826" alt="image" src="https://github.com/community/community/assets/12762439/4f5f33fa-a66f-4770-99fe-98f32b7da450">

Of course the author should take care of the lane length by following coding guidelines but in exploratory settings, people just write and exchange code.

Here some real world example: https://github.com/kopytjuk/python-fmcw/issues/1#issuecomment-1834284210

**Comment by Morriz** [comment]
This is a serious issue. I would love to see this reported as a bug and not some fringe discussion topic...how do we report this for real?
